backend/src/test/java/codezap/template/repository/ThumbnailRepositoryTest.java

```diff
@@ -0,0 +1,131 @@
+package codezap.template.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import codezap.category.domain.Category;
+import codezap.category.repository.CategoryRepository;
+import codezap.global.exception.CodeZapException;
+import codezap.global.repository.JpaRepositoryTest;
+import codezap.member.domain.Member;
+import codezap.member.repository.MemberRepository;
+import codezap.template.domain.SourceCode;
+import codezap.template.domain.Template;
+import codezap.template.domain.Thumbnail;
+
+@JpaRepositoryTest
+public class ThumbnailRepositoryTest {
+
+    @Autowired
+    private MemberRepository memberRepository;
+
+    @Autowired
+    private CategoryRepository categoryRepository;
+
+    @Autowired
+    private TemplateRepository templateRepository;
+
+    @Autowired
+    private SourceCodeRepository sourceCodeRepository;
+
+    @Autowired
+    private ThumbnailRepository sut;
+
+    @Nested
+    @DisplayName("id로 썸네일 조회")
+    class FetchById {
+
+        @Test
+        @DisplayName("성공: id로 썸네일 조회")
+        void fetchByIdSuccess() {
+            // given
+            var member = new Member("Zappy", "password", "salt");
+            memberRepository.save(member);
+            var category = Category.createDefaultCategory(member);
+            categoryRepository.save(category);
+            var template = new Template(member, "Template Title", "Description", category);
+            templateRepository.save(template);
+            var sourceCode = new SourceCode(template, "filename", "content", 1);
+            sourceCodeRepository.save(sourceCode);
+            var thumbnail = new Thumbnail(template, sourceCode);
```

Comment:
```suggestion
            var member = memberRepository.save(new Member("Zappy", "password", "salt"));
            var category = categoryRepository.save(Category.createDefaultCategory(member));
            var template = templateRepository.save(new Template(member, "Template Title", "Description", category));
            var sourceCode = sourceCodeRepository.save(new SourceCode(template, "filename", "content", 1));
```
[제안]
Repository 에 저장되기 전의 객체들은 id 값을 가지기 이전이니 변수로 추출할 필요는 없어 보여요.
그렇다고 변수에 재할당을 할 필요도 없어 보여서 각각을 한 줄로 줄여 보았어요.
가볍게 읽어 보시고 제우스의 취향에 맞게 선택해 주시면 좋을 것 같아요!

Reply:
저도 짱수 제안에 한발 올려봅니당...ㅎㅎ